.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted package name to the issue description
+
+**Packages impacted by this PR:**
+
+
+**Issues associated with this PR:**
+
+
+**Describe the problem that is addressed by this PR:**
```

Comment:
This the most important thing here in my opinion as it answers the `why` - what do you think about making this more prominent? Maybe this template should be changed into markdown headers and this is at the top? One thing I see a lot is:

## Problem

## Solution

## Details / callouts / the rest of the items

(although I personally use what and why)